### DISCUSSION TITLE
How to close the "PORTS" panel?

### DISCUSSION THREAD

**Discussion by wanggaoteng** [microsoft/vscode-discussions#878] [Extension Development QnA]
This has puzzled me for a long time, is there any way to close the "PORTS" panel, as the picture shows:
![1](https://github.com/microsoft/vscode-discussions/assets/29191854/01a92a56-2f08-4f6c-9b0f-2618727fdb82)

**Comment by gjsjohnmurray** [comment]
Please see this pinned post for guidance about the purpose of this discussion forum, and for pointers to other places more suitable for your question.

https://github.com/microsoft/vscode-discussions/discussions/5

**Comment by wanggaoteng** [comment]
> ###
> [gjsjohnmurray ](/gjsjohnmurray) [ Oct 21, 2023 ](#discussioncomment-7346787)
> -

Hi, sorry for a wrong place to ask the question. 
![image](https://github.com/microsoft/vscode-discussions/assets/29191854/cdfd483f-662a-4f0d-972f-e3c1323ae7b1)
